homeassistant/components/mqtt/config_flow.py

```diff
@@ -540,12 +529,10 @@ def validate_user_input(
     data_schema_fields: dict[str, PlatformField],
     errors: dict[str, str],
```

Comment:
I think this can be removed, and we let the function instead return `(config, errors)`?

Reply:
done